python/housinginsights/sources/wmata_distcalc.py

```diff
@@ -133,10 +134,11 @@ def get_dist(self, unique_data_ids=None, sample=False, output_type='csv',db='loc
                 self._array_to_csv(self.distHeader, self.distOutput, self.output_paths[u])
```

Comment:
Ya, the _array_to_csv expects a different format (list of lists, where each inner list is a row i.e. `[[a,b,c],[x,y,z]`)  from result_to_csv (json, i.e. '[{'col1':'a','col2':'b','col3':'c'},{'col1':'x','col2':'y','col3':'z'}]`). We should probably combine all these into one method (or several) and put them all on the base class.